RenderStatePackager/include/ParsingEnvironment.hpp

```diff
@@ -66,11 +65,11 @@ class ParsingEnvironment final
 
     IThreadPool* GetThreadPool();
 
-    bool Initilize();
+    bool Initialize();
 
     ParsingEnvironment(const ParsingEnvironmentCreateInfo& CI);
 
-    ~ParsingEnvironment();
+    ~ParsingEnvironment() = default;
```

Comment:
There is usually no need for defaulting the destructor as it will always be generated.